src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,129 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node<>(tail, value, null);
```

Comment:
Create constructor for this:
```suggestion
        Node<T> newNode = new Node<>(tail, value);
```